scripts/supercloud/run_random_query_experiments.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
so this is the script that you used to determine the 0.03 threshold, but we're not gonna use this script regularly going forward right? (this is fine and good to include still, just checking)

Reply:
correct. also i verified that we can calculate the appropriate query probability based off of the environment and how many queries the main approach makes!